roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -64,6 +64,28 @@
     name: 'install_yamls_makes'
     tasks_from: 'make_edpm_baremetal_compute'
 
+- name: Pass through extra nova-compute config
+  when: cifmw_edpm_deploy_baremetal_nova_compute_extra_config is defined
```

Comment:
question(blocking): Seems this when clause is always going to pass due to cifmw_edpm_deploy_baremetal_nova_compute_extra_config being set by default being set to ""

Is this what you intend? I would have expected something like: 

```
- name: Pass through extra nova-compute config
  when: 
    - cifmw_edpm_deploy_baremetal_nova_compute_extra_config is defined
    - cifmw_edpm_deploy_baremetal_nova_compute_extra_config | length > 0
 ```

Reply:
Yes right, can avoid running it always, the end result is empty conf so was kind of noop, but having this skipped should be better. Will update, thx.